services/integrate-android-app-with-custom-service.adoc

```diff
@@ -0,0 +1,70 @@
+= Integrate an Android Application with Custom Services Running in OpenShift
```

Comment:
Creating an Android App that calls an AeroGear Mobile Custom Service?
@darahayes  trying to frame this in a context that a user would want to achieve

Reply:
@finp I'm not a fan of the title myself but it wouldn't be right to say 'AeroGear Mobile Custom Service.' 

In laymans terms this document shows 'How to connect my app to my backend service using the Aerogear SDK'

We're talking about how we can integrate an Android App to a backend service written, maintained and owned by the end user such that they can access this service seamlessly using the Aerogear Android SDK.

I think 'AeroGear Mobile Custom Service.' is not the right terminology because the service has nothing to do with Aerogear and the service isn't necessarily mobile focused.

Reply:
k, let me digest that, thx ;)